content/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization.md

```diff
@@ -87,9 +87,9 @@ You can only choose an additional permission if it's not already included in the
 
 ### Security
 
-- **Read {% data variables.product.prodname_code_scanning %} results**
-- **Write {% data variables.product.prodname_code_scanning %} results**
-- **Delete {% data variables.product.prodname_code_scanning %} results**
+- **Read {% data variables.product.prodname_code_scanning %} results**: Provide users read permissions for code scanning alerts
```

Comment:
Replacing `code scanning` with its variable and adding a full stop so that this is consistent with the rest of the article:
```suggestion
- **Read {% data variables.product.prodname_code_scanning %} results**: Provide users read permissions for {% data variables.product.prodname_code_scanning %} alerts.
```